docs/content/reference/konnectivity.md

```diff
@@ -0,0 +1,393 @@
+    the data plane: This is the case for services such as webhooks and
+    aggregated API servers that run in the data plane, as well as OLM
+    catalogs, or network checks. For this use case, customer-configured
+    HTTP/S proxies are not relevant, because traffic is only needed to
+    reach inside the data plane. The socks5 proxy is adequate for this.
+
+-   Routing of traffic from the control plane to destinations outside
+    the data plane but potentially only accessible from the data plane.
+    This is the case for ImageStream registries, and external identity
+    providers. For this use case, customer-configured HTTP/S proxy
+    configuration should be taken into consideration because traffic may
+    only be able to leave the data plane through those proxies. For this
+    use case, the HTTP proxy is needed.
+
+## Konnectivity and Control Plane Aggregated API Servers
+
+For OpenShift clusters servers such as openshift-apiserver,
+openshift-oauth-apiserver, and OLM packageserver are aggregated API
+servers that handle essential OpenShift resources such as ImageStreams,
+Users, and Groups.
+
+Because we consider these servers part of the control plane, they run in
+the control plane namespace along with the Kubernetes API server. They
+are configured via APIService resources which tell the Kubernetes API
+server how to reach them to serve up their resources. APIServices have
+the limitation of only allowing a Kubernetes Service to be configured as
+the endpoint of the APIService. This means that a Service resource must
+exist in the data plane for every APIService we want to configure for
+the HostedCluster. This Service in the data plane needs to send traffic
+to the components in the control plane.
+
+The way this is accomplished is that a Service without pod selector is
+created in the default namespace. Then a corresponding Endpoints
+resource is created in the same namespace that contains the IP of the
+service in the control plane namespace. Whenever the Kubernetes API
+Server wants to reach the aggregated API server, it ends up sending
+traffic to the service in the control plane.
+
+One problem with the way Konnectivity is configured is that when you
+configure the egress selector for the cluster to be proxied through
+Konnectivity, then all the traffic destined for services is sent through
+Konnectivity. For these services that live in the control plane
+namespace, the Konnectivity server needs to know to send traffic to the
+control plane and not the data plane.
+
+The way that the Konnectivity server can send traffic to the control
+plane and not the data plane, is that we run a Konnectivity agent pod in
+the control plane. This agent is configured with the --agent-identifiers
+flag to only accept traffic destined for the control plane aggregated
+API services. This means that only traffic that is destined for those
+services can be routed through the Konnectivity server.
```

Comment:
s/accept/route/

s/can be routed through the Konnectivity server/will be routed to the control plane through the Konnectivity agent/